opencti-platform/opencti-front/vitest.config.ts

```diff
@@ -1,13 +1,19 @@
 import { defineConfig } from 'vitest/config';
 import react from '@vitejs/plugin-react';
 import relay from "vite-plugin-relay";
+import * as path from "node:path";
 
 export default defineConfig({
   plugins: [react(), relay],
   test: {
     environment: 'jsdom',
     setupFiles: './setup-vitest.ts',
-    include: ['src/**/**/*.test.{ts,tsx}'],
+    include: ['src/**/*.test.{ts,tsx}'],
```

Comment:
why this change in the config?

Reply:
In testing context, the alias resolution was failing in a specific test file.
This is an oversight from my attempts to fix it.